hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystemV1.java

```diff
@@ -475,6 +475,15 @@ public void testFileSystem() throws Exception {
 
     testSeekOnFileLength();
     tableCleanup();
+
+    testFileDelete();
+    tableCleanup();
+
+    testDeleteRoot();
+    tableCleanup();
+
+    testRecursiveDelete();
+    tableCleanup();
   }
 
   /**
```

Comment:
Can we rename tableCleanup to deleteRootDir or something meaningful?
And also update Javadoc now delete is supported for V1 with this patch.

Reply:
Done in latest commit.